### PR TITLE
feat(docs): Add eslint and prettier GH Actions.

### PR DESCRIPTION
**Pull request by petejohanson**
Adds Prettier and ESLint checks to our docs.

### CODE REVIEW

**Comment by innovaker** [comment]
How does one test this before it reaches `main`?

**Comment by petejohanson** [comment]
@innovaker Let me branch this branch, break it, and try to open a PR into this branch. Should show it in action.

File: docs/package.json [hunk 1]
```diff
@@ -8,7 +8,10 @@
     "serve": "docusaurus serve",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
-    "clear": "docusaurus clear"
+    "clear": "docusaurus clear",
+    "lint": "eslint . --ext js,jsx,md,mdx",
+    "prettier:check": "prettier --check .",
+    "prettier:format": "prettier --write ."
```
Comment (innovaker) [inline docs/package.json:14]: I've been looking around at other examples / best practices.
Perhaps just `check` and `format`?

**Review by innovaker — COMMENTED** [review]
It worked locally.

File: docs/package.json (hunk 1 above)
Comment (innovaker) [reply docs/package.json:14]: Then again, maybe it's fine.

Comment (innovaker) [reply docs/package.json:14]: There seems to be different conventions depending where you go.

Comment (innovaker) [reply docs/package.json:14]: I retract my suggestion.

**Comment by petejohanson** [comment]
@innovaker Failing PR: https://github.com/petejohanson/zmk/pull/1

**Comment by innovaker** [comment]
LGTM (as far as I can tell)

**Pull request merged**